app/controllers/admin/users_controller.rb

```diff
@@ -18,8 +18,8 @@ def edit
 
   def update
     @user = User.find(params[:id])
-    @user.update!(user_params)
```

Comment:
This is calling update! with a bang, which means if it fails, it should error, and therefore never proceed. Therefore this if does nothing.

Reply:
Will remove